beacon-chain/blockchain/log.go

```diff
@@ -88,10 +96,6 @@ func logBlockSyncStatus(block interfaces.ReadOnlyBeaconBlock, blockRoot [32]byte
 			"chainServiceProcessedTime": prysmTime.Now().Sub(receivedTime),
 			"deposits":                  len(block.Body().Deposits()),
 		}
-		commits, err := block.Body().BlobKzgCommitments()
-		if err == nil {
-			lf["commitments"] = len(commits)
-		}
```

Comment:
We should either log the field count in either `Synced new block` or `Finished applying state transition`. It seems to me `Finished applying state transition` is more appropriate because there's where we log attestation, slashing, exit counts... etc